release/internal/config/operator.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/projectcalico/calico/release/internal/command"
+	"github.com/projectcalico/calico/release/internal/version"
+)
+
+type OperatorConfig struct {
+	// GitRemote is the remote for the git repository
+	GitRemote string `envconfig:"OPERATOR_GIT_REMOTE" default:"origin"`
+
+	// Organization is the GitHub organization for the operator
+	Organization string `envconfig:"OPERATOR_GIT_ORGANIZATION" default:"tigera"`
+
+	// GitRepository is the repository for the operator
+	GitRepository string `envconfig:"OPERATOR_GIT_REPOSITORY" default:"operator"`
+
+	// Branch is the repository for the operator
+	Branch string `envconfig:"OPERATOR_BRANCH" default:"master"`
+
+	// RepoReleaseBranchPrefix is the prefix for the release branch
+	RepoReleaseBranchPrefix string `envconfig:"OPERATOR_RELEASE_BRANCH_PREFIX" default:"release"`
+
+	// DevTagSuffix is the suffix for the development tag
+	DevTagSuffix string `envconfig:"OPERATOR_DEV_TAG_SUFFIX" default:"0.dev"`
```

Comment:
this is a bit of a port from process, i will like to keep it for this PR (and remove later in a separate PR) since there are some inconsistencies in the value